### DISCUSSION TITLE
Volume button + next - previous song while screen is off

### DISCUSSION THREAD

**Discussion by gaining** [libre-tube/LibreTube#5337] [Ideas]
### Steps to reproduce

Play music in audio only mode from any youtube playlist example: [music playlist](https://www.youtube.com/playlist?list=PLN4MwbWkCSmXFehtW0zhcqhRgE6Dtjg2b) turn screen off. Play a song, then press on volume + for next song or - for previous song and that doesn't work. This works on other music players, soundcloud app and also youtube but for some reason that's broken in this release

By the way, I submitted a bug about not being able to play the next song of a playlist and that is fixed in release 0.20.0. Thanks for this!

### Expected behavior

Being able to go to next or previous songs while pressing vol +/- button while screen is off.

### Actual behavior

stays on same song, nothing happens.

### LibreTube version

0.20.0

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
That's a feature request, not a bug...

**Comment by Pamilg8** [comment]
Wouldn't it be nice to implement this function as a double tap on the device buttons?
